noir-projects/aztec-nr/aztec/src/context/public_context.nr

```diff
@@ -70,6 +70,10 @@ impl PublicContext {
         }
     }
 
+    pub fn assert_deployment(self) {
+        // TODO(#4738): Implement this
```

Comment:
Same as above

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/4739/commits/e43fade1762b2d3cab5ea400f73111496df52ba0